tests/base_test.py

```diff
@@ -411,22 +411,35 @@ def test_queries_with_rule_buffertime(ea):
         mock_es_init.return_value = mock_es
         ea.run_rule(ea.rules[0], END, START)
 
-    # Assert that es.search is run against every run_every timeframe between START and END
+    # Assert that es.search is run against every buffer_time timeframe between START and END
```

Comment:
My suggestion here would be to extract this to a method and provide a nice long name rather than a comment ;)